app/main.py

```diff
@@ -2,0 +1,32 @@
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.health = health
+        self.name = name
+        self.hidden = False
+
+        self.alive.append(self)
+
+    def die(self) -> None:
+        if self.health <= 0:
+            Animal.alive.remove(self)
```

Comment:
Ensure that `Animal.alive.remove(self)` is used to remove the instance from the class-level list when the animal dies. This is correct as it stands.